all-spark-notebook/Dockerfile

```
@@ -23,7 +23,7 @@ USER $NB_UID
 
 # R packages
 RUN conda install --quiet --yes \
-    'r-base=3.6.3' \
+    'r-base=4.0*' \
```

You should specify 3 digits for the `r-base` version, since the last digit can bring major changes.
```suggestion
    'r-base=4.0.0' \
```